src/app/services/index.js

```diff
@@ -192,24 +187,36 @@ async function loadServiceHistoryFiles(serviceId) {
   };
 }
 
-export async function getIdsOfModified() {
-  const __dirname = path.dirname(new URL(import.meta.url).pathname);
-  const rootPath = path.join(__dirname, '../../../');
-
-  const git = simpleGit(rootPath, { maxConcurrentProcesses: 1 });
-  const committedFiles = await git.diff([ '--name-only', 'master...HEAD', '--', 'services/*.json' ]);
-  const status = await git.status();
-  const modifiedFiles = [
-    ...status.not_added, // Files created but not already in staged area
-    ...status.modified, // Files modified
-    ...status.created, // Files created and in the staged area
-    ...status.renamed.map(({ to }) => to), // Files renamed
-    ...committedFiles.trim().split('\n'), // Files committed
-  ];
-
-  return modifiedFiles
-    .filter(fileName => fileName.match(/services.*\.json/) && !fileName.includes('.history.json'))
-    .map(filePath => path.basename(filePath, '.json'));
+export function getServicesDeclarationsPath() {
+  let serviceDeclarationsPath = path.resolve(
+    process.cwd(),
+    './declarations',
+  );
+
+  if (config.has('services.declarationsPath')) {
+    serviceDeclarationsPath = path.resolve(
+      process.cwd(),
```

Comment:
Relying on `cwd` seems very brittle to me. It is also ambiguous for the `services.declarationsPath`: it means it has to be given relative to `cwd`, whereas I would rather expect relative to the package base directory.